src/cartridges/int_adyen_overlay/cartridge/scripts/adyenCheckout.js

```diff
@@ -112,6 +113,19 @@ function createPaymentRequest(args) {
       args.addTaxPercentage = true;
       if(paymentRequest.paymentMethod.type.indexOf('klarna') > -1){
```

Comment:
are we using formatting like prettier or something?

Reply:
also maybe in future this payment methods can be constants instead of using as strings, but not a big thing now.